src/Providers/Amazon.Sagemaker/test/SageMakerTests.cs

```diff
@@ -2,33 +2,64 @@
 using LangChain.Chains.Sequentials;
 using LangChain.Prompts;
 using LangChain.Schema;
+using System.Text.Json.Nodes;
 using static LangChain.Chains.Chain;
 
 namespace LangChain.Providers.Amazon.SageMaker.Tests;
 
 [TestFixture, Explicit]
 public class SageMakerTests
 {
+    static async Task<string?> DeserializeFunc(HttpResponseMessage response)
```

Comment:
Can we move this as a default feature for SageMakerSettings?
Still, it seems to me that it will be difficult for the user to set this up and we need to try to provide as many implementations of this as possible for the main models used, leaving this setting for something specific

Reply:
good idea